easybuild/easyconfigs/b/Boost/Boost-1.70.0-CrayGNU-19.06.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'boostcray'
```

Comment:
Is the custom easyblock `boostcray` still needed? 
The other recipes in your pull request use the standard easyblock `Boost`.

Reply:
No, it's possible without `boostcray`

Reply:
in that case I think it is better to avoid using the custom easyblock if not really needed.

Reply:
I removed it